recipe/meta.yaml

```diff
@@ -42,8 +42,8 @@ test:
         - test -f ${PREFIX}/lib/libdf.dylib  # [osx]
         - test -f ${PREFIX}/lib/libmfhdf.so  # [linux]
         - test -f ${PREFIX}/lib/libmfhdf.dylib  # [osx]
-        - conda inspect linkages -n _test hdf4  # [not win]
-        - conda inspect objects -n _test hdf4  # [osx]
+        - conda inspect linkages -p ${PREFIX} hdf4  # [not win]
+        - conda inspect objects -p ${PREFIX} hdf4  # [osx]
```

Comment:
@lbdreyer I believe the safest way is to use `${CONDA_DEFAULT_ENV}` instead. @msarahan can probably confirm that.

PS: there are more changes needed in conda-build-all to get this to pass.